### PR TITLE
Whats new October

### LINKED ISSUE #21131: All "What's new" pages should link to .NET Downloads
All "what's new" pages in the dotnet docs set should have a link to [download the latest .NET SDK](https://dotnet.microsoft.com/download).

The list of pages are:
- Last 6 months of ["what's new"](https://docs.microsoft.com/dotnet/whats-new/) including the landing page
- [What's new in .NET 5](https://docs.microsoft.com/dotnet/core/dotnet-five)
- [What's new in .NET Standard](https://docs.microsoft.com/dotnet/standard/whats-new/whats-new-in-dotnet-standard)
- [What's new in .NET Framework](https://docs.microsoft.com/dotnet/framework/whats-new/)
- [What's new in C#](https://docs.microsoft.com/dotnet/csharp/whats-new/csharp-9)
- [What's new in F#](https://docs.microsoft.com/dotnet/fsharp/whats-new/fsharp-47)
- [What's new in VB](https://docs.microsoft.com/dotnet/visual-basic/whats-new/)

Also, the What's new landing page doesn't link to the first three links listed above. Those should be added as well.

### PR DESCRIPTION
**Pull request by BillWagner**
Create the "what's new" page for October.

Due to the .NET 5 release today, delayed until now and includes all content up to Nov 10.

Also,

Fixes #21131

### CODE REVIEW

File: docs/whats-new/2020-10.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+- [NETSDK1005 and NETSDK1047: Asset file is missing target](../core/tools/sdk-errors/netsdk1005) - Adding .NET SDK error pages for 1005, 1047, and 1059
+- [NETSDK1013: The TargetFramework value was not recognized](../core/tools/sdk-errors/netsdk1013) - Create netsdk1013.md
+- [NETSDK1022: Duplicate items were included.](../core/tools/sdk-errors/netsdk1022) - Create netsdk1022.md
+- [NETSDK1059: Project contains obsolete .NET CLI tool](../core/tools/sdk-errors/netsdk1059) - Adding .NET SDK error pages for 1005, 1047, and 1059
+- [NETSDK1071: Explicitly versioned PackageReference to a metapackage that would be included with the framework.](../core/tools/sdk-errors/netsdk1071) - Create a page for NETSDK1071
+- [NETSDK1073: The FrameworkReference was not recognized](../core/tools/sdk-errors/netsdk1073) - Create netsdk1073.md
+- [NETSDK1079: The Microsoft.AspNetCore.All package is not supported when targeting .NET Core 3.0 or higher.](../core/tools/sdk-errors/netsdk1079) - Add error message documentation for NETSDK1079
+- [Resolve NETSDK1145, targeting or apphost pack missing](../core/tools/sdk-errors/netsdk1145) - Add targeting/runtime pack missing
```
Comment (tdykstra) [inline docs/whats-new/2020-10.md:44]: I suggest we remove these error pages, as they're only of interest when you actually get the particular error message.

**Review by tdykstra — APPROVED** [review]
I added a comment suggesting we remove the NETSDK error pages, but I also wonder if it might be worth doing that with the CAxxxx and IDExxxx pages as well.

File: docs/csharp/whats-new/csharp-9.md [hunk 2]
```diff
@@ -26,6 +26,8 @@ C# 9.0 adds the following features and enhancements to the C# language:
 
 C# 9.0 is supported on **.NET 5**. For more information, see [C# language versioning](../language-reference/configure-language-version.md).
 
+You can download that latest [.NET SDK](https://dotnet.microsoft.com/download) from the .NET downloads page.
```
Comment (gewarren) [inline docs/csharp/whats-new/csharp-9.md:29]:
```suggestion
You can download the latest .NET SDK from the [.NET downloads page](https://dotnet.microsoft.com/download).
```

File: docs/fsharp/whats-new/fsharp-50.md [hunk 3]
```diff
@@ -7,6 +7,8 @@ ms.date: 11/06/2020
 
 F# 5.0 adds several improvements to the F# language and F# Interactive. It is released with **.NET 5**.
 
+You can download that latest [.NET SDK](https://dotnet.microsoft.com/download) from the .NET downloads page.
```
Comment (gewarren) [inline docs/fsharp/whats-new/fsharp-50.md:10]:
```suggestion
You can download the latest .NET SDK from the [.NET downloads page](https://dotnet.microsoft.com/download).
```

File: docs/visual-basic/whats-new/index.md [hunk 4]
```diff
@@ -18,6 +18,8 @@ This topic lists key feature names for each version of Visual Basic, with detail
 Visual Basic 16.0 / Visual Studio 2019 version 16.0\
 For new features, see [Visual Basic 16.0](#visual-basic-160).
 
+You can download that latest [.NET SDK](https://dotnet.microsoft.com/download) from the .NET downloads page.
```
Comment (gewarren) [inline docs/visual-basic/whats-new/index.md:21]:
```suggestion
You can download the latest .NET SDK from the [.NET downloads page](https://dotnet.microsoft.com/download).
```

File: docs/whats-new/2020-06.md [hunk 5]
```diff
@@ -9,6 +9,8 @@ ms.date: 06/30/2020
 
 Welcome to what's new in the .NET docs for June 2020. This article lists some of the major changes to docs during this period.
 
+You can download that latest [.NET SDK](https://dotnet.microsoft.com/download) from the .NET downloads page.
```
Comment (gewarren) [inline docs/whats-new/2020-06.md:12]:
```suggestion
You can download the latest .NET SDK from the [.NET downloads page](https://dotnet.microsoft.com/download).
```

File: docs/whats-new/2020-07.md [hunk 6]
```diff
@@ -8,6 +8,8 @@ ms.date: 08/03/2020
 
 Welcome to what's new in the .NET docs from July 2020. This article lists some of the major changes to docs during this period.
 
+You can download that latest [.NET SDK](https://dotnet.microsoft.com/download) from the .NET downloads page.
```
Comment (gewarren) [inline docs/whats-new/2020-07.md:11]:
```suggestion
You can download the latest .NET SDK from the [.NET downloads page](https://dotnet.microsoft.com/download).
```

File: docs/whats-new/2020-08.md [hunk 7]
```diff
@@ -8,6 +8,8 @@ ms.date: 09/01/2020
 
 Welcome to what's new in the .NET docs from August 1, 2020 through August 31, 2020. This article lists some of the major changes to docs during this period.
 
+You can download that latest [.NET SDK](https://dotnet.microsoft.com/download) from the .NET downloads page.
```
Comment (gewarren) [inline docs/whats-new/2020-08.md:11]:
```suggestion
You can download the latest .NET SDK from the [.NET downloads page](https://dotnet.microsoft.com/download).
```

File: docs/whats-new/2020-09.md [hunk 8]
```diff
@@ -8,6 +8,8 @@ ms.date: 10/02/2020
 
 Welcome to what's new in the .NET docs from September 1, 2020 through September 30, 2020. This article lists some of the major changes to docs during this period.
 
+You can download that latest [.NET SDK](https://dotnet.microsoft.com/download) from the .NET downloads page.
```
Comment (gewarren) [inline docs/whats-new/2020-09.md:11]:
```suggestion
You can download the latest .NET SDK from the [.NET downloads page](https://dotnet.microsoft.com/download).
```

File: docs/whats-new/dotnet-2020-05.md [hunk 9]
```diff
@@ -8,6 +8,8 @@ ms.date: 06/01/2020
 
 Welcome to what's new in .NET docs for May 2020. This article lists some of the major changes to docs during this period.
 
+You can download that latest [.NET SDK](https://dotnet.microsoft.com/download) from the .NET downloads page.
```
Comment (gewarren) [inline docs/whats-new/dotnet-2020-05.md:11]:
```suggestion
You can download the latest .NET SDK from the [.NET downloads page](https://dotnet.microsoft.com/download).
```

File: docs/whats-new/2020-10.md [hunk 10]
```diff
@@ -0,0 +1,258 @@
...
+title: ".NET docs: What's new for October 1, 2020 - November 10, 2020"
+description: "What's new in the .NET docs for October 1, 2020 - November 10, 2020."
+ms.date: 11/10/2020
+---
+
+# .NET docs: What's new for October 1, 2020 - November 10, 2020
+
+Welcome to what's new in the .NET docs from October 1, 2020 through November 10, 2020. This article lists some of the major changes to docs during this period.
+
+You can download that latest [.NET SDK](https://dotnet.microsoft.com/download) from the .NET downloads page.
```
Comment (gewarren) [inline docs/whats-new/2020-10.md:11]:
```suggestion
You can download the latest .NET SDK from the [.NET downloads page](https://dotnet.microsoft.com/download).
```

File: docs/whats-new/2020-10.md [hunk 11]
```diff
@@ -0,0 +1,258 @@
...
+- [NETSDK1022: Duplicate items were included.](../core/tools/sdk-errors/netsdk1022) - Create netsdk1022.md
+- [NETSDK1059: Project contains obsolete .NET CLI tool](../core/tools/sdk-errors/netsdk1059) - Adding .NET SDK error pages for 1005, 1047, and 1059
+- [NETSDK1071: Explicitly versioned PackageReference to a metapackage that would be included with the framework.](../core/tools/sdk-errors/netsdk1071) - Create a page for NETSDK1071
+- [NETSDK1073: The FrameworkReference was not recognized](../core/tools/sdk-errors/netsdk1073) - Create netsdk1073.md
+- [NETSDK1079: The Microsoft.AspNetCore.All package is not supported when targeting .NET Core 3.0 or higher.](../core/tools/sdk-errors/netsdk1079) - Add error message documentation for NETSDK1079
+- [Resolve NETSDK1145, targeting or apphost pack missing](../core/tools/sdk-errors/netsdk1145) - Add targeting/runtime pack missing
+
+### Updated articles
+
+- [SYSLIB0004: The constrained execution region (CER) feature is not supported](../core/compatibility/syslib0004)
```
Comment (gewarren) [inline docs/whats-new/2020-10.md:48]: This appears in the new article section as well, so probably don't include it here.

File: docs/whats-new/2020-10.md [hunk 12]
```diff
@@ -0,0 +1,258 @@
...
+### Updated articles
+
+- [Nullable reference types](../csharp/nullable-references) - Add known pitfalls to nullable reference types
+- [What's new in C# 7.0 through C# 7.3](../csharp/whats-new/csharp-7) - Consolidate c7 whats new
+
+## .NET Framework
+
+### New articles
+
+- [FindHeaderByColumn method](../framework/additional-apis/system.windows.controls.gridviewheaderrowpresenter.findheaderbycolumn) - Document private APIs in GridViewHeaderRowPresenter
```
Comment (gewarren) [inline docs/whats-new/2020-10.md:72]: The first 3 were added for compliance reasons and probably shouldn't be advertised here?

File: docs/whats-new/2020-10.md [hunk 13]
```diff
@@ -0,0 +1,258 @@
...
+- [Extract elements from a string](../standard/base-types/parse-strings) - Move alternatives out of String.Split ref
+- [Behavior changes when comparing strings on .NET 5+](../standard/base-types/string-comparison-net-5-plus) - Behavior changes due to NLS -> ICU switch on Windows
+- [Runtime libraries overview](../standard/runtime-libraries-overview) - Reorg Fundamentals TOC
+
+### Updated articles
+
+- [Compare strings in .NET](../standard/base-types/comparing) - Behavior changes due to NLS -> ICU switch on Windows
+- [Standard date and time format strings](../standard/base-types/standard-date-and-time-format-strings) - Q2 Content performance updates
+- [Managed Execution Process](../standard/managed-execution-process) - .NET Framework -> .NET
+- [Serialization guidelines](../standard/serialization/serialization-guidelines) - .NET Framework -> .NET (/standard/s*)
```
Comment (gewarren) [inline docs/whats-new/2020-10.md:206]:
```suggestion
- [Serialization guidelines](../standard/serialization/serialization-guidelines) - .NET Framework -> .NET
```

**Review by gewarren — COMMENTED** [review]
Do we not add redirects for deleted what's new pages?

**Comment by tdykstra** [comment]
>Do we not add redirects for deleted what's new pages?

Given the clear limitations on the time frames for when they're useful, I wouldn't.

File: docs/whats-new/2020-10.md [hunk 14]
```diff
@@ -0,0 +1,258 @@
...
+
+- [with expression (C# reference)](../csharp/language-reference/operators/with-expression) - C# 9 reference update: a with expression
+- [Use pattern matching to build your class behavior for better code](../csharp/tutorials/exploration/patterns-objects) - Pattern tutorial
+- [Tutorial: Explore ideas using top-level statements to build code as you learn](../csharp/tutorials/exploration/top-level-statements) - Top level statements tutorial
+- [Object-Oriented programming (C#)](../csharp/tutorials/intro-to-csharp/object-oriented-programming) - move OO tutorial to get-started
+
+### Updated articles
+
+- [Nullable reference types](../csharp/nullable-references) - Add known pitfalls to nullable reference types
+- [What's new in C# 7.0 through C# 7.3](../csharp/whats-new/csharp-7) - Consolidate c7 whats new
```
Comment (pkulikov) [inline docs/whats-new/2020-10.md:66]:
```suggestion
- [What's new in C# 7.0 through C# 7.3](../csharp/whats-new/csharp-7) - Consolidate C# 7 what's new
```

**Comment by gewarren** [comment]
@BillWagner it looks like many of these links are invalid (build warnings).

**Pull request merged**